packages/fields/types/Content/views/blocks/caption/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Making sure that _every_ block type is bundled, even the dependencies, in case they're not imported directly.